pyment/docstring.py

```diff
@@ -1965,15 +1974,15 @@ def _set_raw_params(self, sep):
                         raw += ' (Default value = ' + str(p[3]) + ')'
                 raw += '\n'
         elif self.dst.style['out'] == 'google':
-            spaces = ' ' * 2
+            spaces = ' ' * self.num_of_spaces
             with_space = lambda s: '\n'.join([self.docs['out']['spaces'] +\
                                                     l.lstrip() if i > 0 else\
                                                     l for i, l in enumerate(s.splitlines())])
             raw += self.dst.googledoc.get_key_section_header('param', self.docs['out']['spaces'])
             for p in self.docs['out']['params']:
                 raw += self.docs['out']['spaces'] + spaces + p[0]
                 if p[2] is not None and len(p[2]) > 0:
-                    raw += '(' + p[2]
+                    raw += ' (' + p[2]
```

Comment:
It is not quite clear to me why you added this space. Would you be willing to briefly explain (as a response to this comment)?

Reply:
This only happens when the input docstring has a type associated with the parameter in the function. And we want to output to Google docstring. Take this function with this reStructuredText docstring:
```python
def example(a):
    """Example Function

    :param a: A value
    :type a: int
    """
    pass
```

With out this added space and running this command: `pyment -i reST -o google -t -w path/to/file.py`

It will output: 

```
def example(a):
    """Example Function

    Args:
        a(int): A value

    Returns:

    Raises:

    """
    pass
```

With no space between "a" and "(int)". Adding this space, adds a space between so it results in: 

```
def example(a):
    """Example Function

    Args:
        a (int): A value

    Returns:

    Raises:

    """
    pass
```

Reply:
Thank you for your elaborate, clear and detailed explanation!